hoot-services/src/main/java/hoot/services/HootProperties.java

```diff
@@ -271,6 +272,9 @@
         // Root folder of tomcat writable locations
         USERFILES_FOLDER = HOME_FOLDER + File.separator + "userfiles";
 
+        // User generated changesets location
+        CHANGESETS_FOLDER = USERFILES_FOLDER + File.separator + "changesets";
+
```

Comment:
moved changeset output from tmp to changesets so they don't get cleaned up when tomcat restarts